src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -393,40 +407,68 @@ subroutine get_fv3_regional_ensperts_run(this,en_perts,nelen,ps_bar)
            if( .not. parallelization_over_ensmembers )then
               if (mype == 0) write(6,'(a,a)') &
                  'CALL READ_FV3_REGIONAL_ENSPERTS FOR ENS DATA with the filename str : ',trim(ensfilenam_str)
+              if (if_model_fed) then
```

Comment:
@hongli-wang  I have a question here.  Why not implementing ,for fed Ensemble,  the  parallel IO options currently used in RRFS.  The comparison had been discussed in this PR . https://github.com/NOAA-EMC/GSI/pull/504.   For the current RRFS (on North American domain,  the current option is faster than the  parallelization_over_ensmembers as discussed above. For convenience, the fig attached here. 
![image](https://github.com/NOAA-EMC/GSI/assets/63461756/5fa2a55a-c46d-4bf8-9be8-cba44b48ad66)
Please convince me that the savings in development effort from implementing these options justify losing the faster options currently used in the RRFS. (@hu5970 please correct me if I am wrong on current RRFS options between the two parallel IO functions).

Reply:
@TingLei-NOAA 
I dont know the background you mentioned. I have assumed parallelization_over_ensmembers is faster than the default. However, it turned out this is not always the case. 
The default option was in this PR, but I removed them in the latest commits. I can have them back.

Reply:
@hongli-wang  Thanks for the clarification.  I am very happy :) that it could be resolved so straightforwardly  :).